src/Microsoft.AspNet.Mvc.Razor/LanguageViewLocationExpander.cs

```diff
@@ -62,7 +59,6 @@ public void PopulateValues(ViewLocationExpanderContext context)
             foreach (var location in viewLocations)
             {
                 yield return location.Replace("{0}", value + "/{0}");
-                yield return location;
```

Comment:
Removing the original is a bit problematic though - you'll never fall back to Home/Index.cshtml if Home/en-us/Index.cshtml is available.